### PR TITLE
Fixed https connection to ElasticSearch

### LINKED ISSUE #37: Connecting to ES using SSL is not working
Need to refactor _connect_to_es, when using SSL without --sslskipverify parameter, connection is done using HTTP, not HTTPs.

Also it seems that using --sslskipverify is also failing

```
$ python3 osde2e-scale-wrapper/osde2e-wrapper.py --account-config /home/morenod/osd/custom_config.yaml --cluster-count 2 -s search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com -p 443 --es-index-only --path /tmp/2dcf88ca-4f98-4242-8785-da48aaec823b --index-retry 3 --log-level info
2021-01-11 13:01:27.821 INFO osde2e-wrapper - main: Logging to console
2021-01-11 13:01:27.821 INFO osde2e-wrapper - main: Starting to upload metadata files to elastic
2021-01-11 13:01:27.822 INFO osde2e-wrapper - main: Attempting to load metadata json /tmp/2dcf88ca-4f98-4242-8785-da48aaec823b/10/metadata.json
2021-01-11 13:01:27.822 INFO osde2e-wrapper - _index_result: Attempting to upload (Attempt: 0) information to ES server with index osde2e-install-timings
2021-01-11 13:01:28.538 WARNING base - log_request_fail: POST http://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com:443/osde2e-install-timings/_doc [status:400 request:0.715s]
2021-01-11 13:01:28.538 WARNING base - _raise_error: Undecodable raw error response from server: Expecting value: line 1 column 1 (char 0)
2021-01-11 13:01:28.539 ERROR osde2e-wrapper - _index_result: RequestError(400, '<html>\r\n<head><title>400 The plain HTTP request was sent to HTTPS port</title></head>\r\n<body>\r\n<center><h1>400 Bad Request</h1></center>\r\n<center>The plain HTTP request was sent to HTTPS port</center>\r\n</body>\r\n</html>\r\n')
2021-01-11 13:01:28.539 ERROR osde2e-wrapper - _index_result: Failed to upload to ES, waiting 5 seconds for next upload retry
2021-01-11 13:01:33.544 INFO osde2e-wrapper - _index_result: Attempting to upload (Attempt: 1) information to ES server with index osde2e-install-timings
2021-01-11 13:01:33.989 WARNING base - log_request_fail: POST http://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com:443/osde2e-install-timings/_doc [status:400 request:0.444s]
2021-01-11 13:01:33.990 WARNING base - _raise_error: Undecodable raw error response from server: Expecting value: line 1 column 1 (char 0)
2021-01-11 13:01:33.990 ERROR osde2e-wrapper - _index_result: RequestError(400, '<html>\r\n<head><title>400 The plain HTTP request was sent to HTTPS port</title></head>\r\n<body>\r\n<center><h1>400 Bad Request</h1></center>\r\n<center>The plain HTTP request was sent to HTTPS port</center>\r\n</body>\r\n</html>\r\n')
2021-01-11 13:01:33.990 ERROR osde2e-wrapper - _index_result: Failed to upload to ES, waiting 10 seconds for next upload retry
```

```
$ python3 osde2e-scale-wrapper/osde2e-wrapper.py --account-config /home/morenod/osd/custom_config.yaml --cluster-count 2 -s search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com -p 443 --es-index-only --path /tmp/2dcf88ca-4f98-4242-8785-da48aaec823b --index-retry 3 --log-level info --sslskipverify True
2021-01-11 13:02:09.555 INFO osde2e-wrapper - main: Logging to console
2021-01-11 13:02:09.556 INFO osde2e-wrapper - main: Starting to upload metadata files to elastic
2021-01-11 13:02:09.557 INFO osde2e-wrapper - main: Attempting to load metadata json /tmp/2dcf88ca-4f98-4242-8785-da48aaec823b/10/metadata.json
2021-01-11 13:02:09.557 INFO osde2e-wrapper - _index_result: Attempting to upload (Attempt: 0) information to ES server with index osde2e-install-timings
2021-01-11 13:02:10.019 WARNING base - log_request_fail: POST http://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com:443/osde2e-install-timings/_doc [status:400 request:0.462s]
2021-01-11 13:02:10.020 WARNING base - _raise_error: Undecodable raw error response from server: Expecting value: line 1 column 1 (char 0)
2021-01-11 13:02:10.020 ERROR osde2e-wrapper - _index_result: RequestError(400, '<html>\r\n<head><title>400 The plain HTTP request was sent to HTTPS port</title></head>\r\n<body>\r\n<center><h1>400 Bad Request</h1></center>\r\n<center>The plain HTTP request was sent to HTTPS port</center>\r\n</body>\r\n</html>\r\n')
2021-01-11 13:02:10.021 ERROR osde2e-wrapper - _index_result: Failed to upload to ES, waiting 5 seconds for next upload retry
```

```
$ python3 osde2e-scale-wrapper/osde2e-wrapper.py --account-config /home/morenod/osd/custom_config.yaml --cluster-count 2 -s search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com -p 443 --es-index-only --path /tmp/2dcf88ca-4f98-4242-8785-da48aaec823b --index-retry 3 --log-level info --sslskipverify False
2021-01-11 13:02:18.399 INFO osde2e-wrapper - main: Logging to console
2021-01-11 13:02:18.399 INFO osde2e-wrapper - main: Starting to upload metadata files to elastic
2021-01-11 13:02:18.400 INFO osde2e-wrapper - main: Attempting to load metadata json /tmp/2dcf88ca-4f98-4242-8785-da48aaec823b/10/metadata.json
2021-01-11 13:02:18.400 INFO osde2e-wrapper - _index_result: Attempting to upload (Attempt: 0) information to ES server with index osde2e-install-timings
2021-01-11 13:02:18.835 WARNING base - log_request_fail: POST http://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com:443/osde2e-install-timings/_doc [status:400 request:0.434s]
2021-01-11 13:02:18.835 WARNING base - _raise_error: Undecodable raw error response from server: Expecting value: line 1 column 1 (char 0)
2021-01-11 13:02:18.836 ERROR osde2e-wrapper - _index_result: RequestError(400, '<html>\r\n<head><title>400 The plain HTTP request was sent to HTTPS port</title></head>\r\n<body>\r\n<center><h1>400 Bad Request</h1></center>\r\n<center>The plain HTTP request was sent to HTTPS port</center>\r\n</body>\r\n</html>\r\n')
2021-01-11 13:02:18.836 ERROR osde2e-wrapper - _index_result: Failed to upload to ES, waiting 5 seconds for next upload retry
```

### PR DESCRIPTION
**Pull request by morenod**
There was a problem connecting to ES using https

I have changed parameters from --server and --port to --es-connection, which will include protocol http(s):// + host + (:port)

I have also renamed related ES arguments to --es- to quickly identify them as ES related

NOTE: To test this PR I have used code from https://github.com/cloud-bulldozer/osde2e-scale-wrapper/pull/36 and https://github.com/cloud-bulldozer/osde2e-scale-wrapper/pull/33, merge them before this one

### CODE REVIEW

**Comment by morenod** [comment]
@dry923 ready to review

File: README.md [hunk 1]
```diff
@@ -47,12 +47,10 @@ without uploading any information**
 
 | Option | Description | Default |
 |--------|-------------|---------|
-| --server | The elasticsearch server| -- |
-| --port | The elasticsearch port | -- |
-| --sslskipverify | If elasticsearch is setup with ssl we can disable tls cert verification. | False |
-| --index | The index to write to. | osde2e-install-timings |
-| --index-retry | Number of retries to connect to ES | 5 |
-
+| --es-connection | The elasticsearch server URL, including protocol and port (if required)| -- |
```
Comment (dry923) [inline README.md]: Thoughts on renaming this es-url to make it more obvious

File: README.md [hunk 2]
```diff
@@ -47,12 +47,10 @@ without uploading any information**
...
+| --es-insecure | sslskipverify | If ES is setup with ssl, but can disable tls cert verification | False |
```
Comment (dry923) [inline README.md]: you have the old sslskipverify still in the line

File: osde2e-wrapper.py [hunk 3]
```diff
@@ -147,6 +149,7 @@ def _verify_cmnd(osde2e_cmnd,my_path):
 
     return osde2e_cmnd
 
+
```
Comment (dry923) [inline osde2e-wrapper.py]: extra line

File: osde2e-wrapper.py [hunk 4]
```diff
@@ -169,6 +172,7 @@ def _download_kubeconfig(osde2ectl_cmd,my_path):
         logging.error(stdout)
         logging.error(stderr)
 
+
```
Comment (dry923) [inline osde2e-wrapper.py]: extra line

File: osde2e-wrapper.py [hunk 5]
```diff
@@ -221,6 +225,7 @@ def _build_cluster(osde2e_cmnd,osde2ectl_cmd,account_config,my_path,es,index,my_
             metadata["timestamp"] = time.strftime("%Y-%m-%dT%H:%M:%S")
             _index_result(es,index,metadata,index_retry)
 
+
```
Comment (dry923) [inline osde2e-wrapper.py]: extra line

**Pull request merged**